go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -167,6 +169,16 @@ func DemystifyPending(status v1.PodStatus) (pluginsCore.PhaseInfo, error) {
 	return pluginsCore.PhaseInfoQueued(time.Now(), pluginsCore.DefaultPhaseVersion, "Scheduling"), nil
 }
 
+func DemystifySuccess(status v1.PodStatus, info pluginsCore.TaskInfo) (pluginsCore.PhaseInfo, error) {
+	for _, status := range status.ContainerStatuses {
```

Comment:
Should we check InitContainer Statuses as well?

Reply:
@EngHabu sure, done